manifests/server.pp

```diff
@@ -1,5 +1,6 @@
-# class to manage the actual prometheus server
-# this is a private class that gets called from the init.pp
+#
+# @summary class to manage the actual prometheus server. This is a private class that gets called from the init.pp
```

Comment:
This class can't actually decide if it's private or not.  README says no, but then there are actually quite a few things you can't set if you don't do it via the main class.

Reply:
correct, I will remove the `private` statement